vpr/SRC/base/vpr_api.c

```diff
@@ -122,6 +122,7 @@ void vpr_print_usage(void) {
 	vpr_printf_info("\t[--td_place_exp_last <float>]\n");
 	vpr_printf_info("\n");
 	vpr_printf_info("Router Options:  [-max_router_iterations <int>] [-bb_factor <int>]\n");
+	vpr_printf_info("\t[--min_incremental_reroute_fanout <int>]\n");
```

Comment:
Since we don't expect this option to be used much, it might make sense to move it down in the print order